src/main/java/org/opensearch/knn/index/codec/KNN990Codec/UnitTestCodec.java

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
I think we are using this simpler form of license header.

```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
I don't think so the header is updated in the repo ref: https://github.com/opensearch-project/k-NN/blob/main/.idea/copyright/SPDX_ALv2.xml. otherwise it would have come automatically from IDE. Let me check my IDE again.

Reply:
@martin-gaievski ?